src/UserManager.ts

```diff
@@ -152,30 +150,52 @@ export class UserManager {
         }
     }
 
-    public async signinSilent(): Promise<User | null> {
+    public async signinSilent(params?: IFrameWindowParams): Promise<User | null> {
```

Comment:
can we rename that argument and  `params` to `args` including the others?
args -> belongs to create request
navigatorParams -> belongs to navigation

Also i guess the type is wrong here, it must be SigninArgs

Reply:
SigninArgs doesn't include the window params that I'm passing here, I haven't dealt with those settings yet. As of this PR, it's not possible to customize those with function arguments. The name `params` was chosen for consistency with the naming with the `NavigatorParams` interface that I've split into `PopupWindowParams`, `RedirectParams`, and `IFrameWindowParams`.